spyder_line_profiler/spyder/widgets.py

```diff
@@ -613,23 +771,74 @@ def populate_tree(self):
 
                 # Monospace font for code
                 line_item.setFont(COL_LINE, monospace_font)
+                
+    def on_item_clicked(self, item):
+        data = item.data(COL_POS, Qt.UserRole)
+        if data is None or len(data)<2: return
+        filename, line_no = data
+        self.sig_edit_goto_requested.emit(filename, line_no, '')
+
 
-    def item_activated(self, item):
-        filename, line_no = item.data(COL_POS, Qt.UserRole)
-        self.edit_goto.emit(filename, line_no, '')
+# =============================================================================
+# Tests
+# =============================================================================
+
+profile = lambda  x:x # dummy profile wrapper to make script load externally
```

Comment:
Some issues with spaces:
```suggestion
profile = lambda x: x  # dummy profile wrapper to make script load externally
```